### PR TITLE
highlight features

### PR DESCRIPTION
**Pull request by pabl0rg**
These small changes try to make features more visible and improve the title tag.

### CODE REVIEW

File: home/index.html [hunk 1]
```diff
@@ -105,6 +101,11 @@ <h2 class="section" id="overview">Features</h2>
 <p>
   This build file also includes a directive to upload your artifacts to Bintray automatically.
 </p>
+  <h2 class="section" id="syntax" indent="1"><span class="glyphicon glyphicon-check"></span> Fast builds</h2>
+  <h2 class="section" id="syntax" indent="1"><span class="glyphicon glyphicon-check"></span> Intuitive plug-in architecture</li></h2>
+  <h2 class="section" id="syntax" indent="1"><span class="glyphicon glyphicon-check"></span> Half the size of Gradle</h2>
```
Comment (cbeust) [inline home/index.html]: I don't think we should say it's half the size of Gradle: users won't care about this and who knows, Kobalt might end up being pretty big too...

File: home/index.html [hunk 2]
```diff
@@ -55,22 +55,18 @@
...
-  <li>Fast builds</li>
-</ul>
-</p>
 
+<h2 class="section" id="features">Features</h2>
+
+<h2 class="section" id="buildFile" indent="1"><span class="glyphicon glyphicon-check"></span> Build file auto-completion in your IDE</h2>
+    <img src="../pics/auto-completion.png" class="img-rounded"/>
+
+<h2 class="section" id="syntax" indent="1"><span class="glyphicon glyphicon-check"></span> Clean syntax for build files</h2>
```
Comment (cbeust) [inline home/index.html]: I took a look at these glyphs and I find them distracting and too big. Can you remove them?

**Comment by cbeust** [comment]
Looks good with just two comments (remove the mention "half the size of gradle" and remove the glyphs).

**Comment by cbeust** [comment]
Actually one more remark (not technically part of this PR): instead of "Fast builds" (which is subjective), we should say "Full support for incremental tasks" (and then maybe add "which leads to faster builds).

File: home/index.html (hunk 1 above)
Comment (pabl0rg) [inline home/index.html]: Yeah, i thought you might not like that.  It's one of the things I like
about Kobalt, esp considering it already includes the kotlin compiler.

But I'll remove it.

Comment (pabl0rg) [inline home/index.html]: Hi Cedric,

I made the suggested changes, I think.

You have more experience but i've read that people don't change their ways
unless the new alternative is 9 times better than the old, hence my concern
for highlighting the advantages.

Best,
Juan

**Comment by cbeust** [comment]
Looks good, thanks @pabl0rg !

**Pull request merged**